clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1151,64 +1151,55 @@ protected String getConfigValue(ConfigKey key, String headerName) {
         }
     }
 
-    public String toHtmlTable() {
-        return toHtmlTable(Collections.<String, String>emptyMap());
-    }
-
-    private void addHeader(StringBuilder builder, String headerName) {
-        builder.append("<th>");
-        builder.append(headerName);
-        builder.append("</th>\n");
-    }
-
-    private void addColumnValue(StringBuilder builder, String value) {
-        builder.append("<td>");
-        builder.append(value);
-        builder.append("</td>");
+    public String toHtmlList() {
```

Comment:
@hachikuji Good point!
In that case, what about keeping `toHtmlTable()` and still calling it from `main()` when no arguments are passed? That way we keep the existing behaviour.
Then in `build.gradle` we can add an argument when invoking `main()` to specify we want to call `toHtmlList()`.

Reply:
I think it's fine to change the output of `main()`. I am mostly considering whether it's worth keeping `toHtmlTable`. This sort of api should not have been exposed in the first place (IMO). Now that it has, probably not much harm keeping it, but should we do the same with `toHtmlList`? An alternative is to have like a `ConfigDefUtils` which is not exposed where we put `toHtmlList`. What do you think?

Reply:
I've tried encapsulating `toHtmlList` into a private `ConfigDefUtils` class but I've had to expose some of the methods in `ConfigDef`: `headers()` and `getConfigvalue()`.
This is mostly because of `LogConfigDef` which overrides some methods. Otherwise we could just move these 2 methods in `ConfigDefUtils`.

I've pushed that attempt so you can see if that what you had in mind.